community-operators/percona-server-mongodb-operator/1.6.0/percona-server-mongodb-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,388 @@
+    are performed using Percona Backup for MongoDB (PBM) and can be stored on
+    local PVs or in any S3-compatible cloud storage provider.
+
+
+    ### Common Configurations
+
+
+    * **Set Member as Arbiter** - [Set up a replica set which contains an
+    arbiter](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/arbiter.html),
+    which participates in elections but does not store any data.  This reduces
+    storage costs while helping maintain replica set integrity.
+
+
+    * **Expose Members Outside K8S** - [by appropriately configuring the
+    ServiceType](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/expose.html)
+    you can expose replica set members outside of Kubernetes or provide
+    statically assigned IP addresses.
+
+
+    * **Utilize Local Storage Options** - [with support for Local Storage you
+    can mount existing data
+    directories](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/storage.html)
+    into your replica set managed by Kubernetes or utilize high performance
+    hardware for local storage rather than network storage for your database.
+
+
+    ### Before You Start
+
+
+    Add the PSMDB user `Secret` to Kubernetes. User information must be placed
+    in the data section of the `secrets.yaml`
+
+    file with Base64-encoded logins and passwords for the user accounts.
+
+
+    Below is a sample `secrets.yaml` file for the correct formatting.
+
+
+    ```
+
+    apiVersion: v1
+
+    kind: Secret
+
+    metadata:
+      name: my-cluster-name-secrets
+    type: Opaque
+
+    data:
+      MONGODB_BACKUP_USER: YmFja3Vw
+      MONGODB_BACKUP_PASSWORD: YmFja3VwMTIzNDU2
+      MONGODB_CLUSTER_ADMIN_USER: Y2x1c3RlckFkbWlu
+      MONGODB_CLUSTER_ADMIN_PASSWORD: Y2x1c3RlckFkbWluMTIzNDU2
+      MONGODB_CLUSTER_MONITOR_USER: Y2x1c3Rlck1vbml0b3I=
+      MONGODB_CLUSTER_MONITOR_PASSWORD: Y2x1c3Rlck1vbml0b3IxMjM0NTY=
+      MONGODB_USER_ADMIN_USER: dXNlckFkbWlu
+      MONGODB_USER_ADMIN_PASSWORD: dXNlckFkbWluMTIzNDU2
+      PMM_SERVER_USER: cG1t
+      PMM_SERVER_PASSWORD: c3VwYXxefHBheno=
+    ```
+
+
+
+    *  Add support for mongos service to expose a single shard of a MongoDB cluster through one entry point instead of provisioning a load-balancer per replica set node. In the following release, we will add support for multiple shards.
+
+    *  Official support for Percona Monitoring and Management (PMM) v.2
+
+    *  Percona Server for MongoDB 4.4 is now supported
+
+    *  OpenShift Container Platform 4.6 is now supported
+
+    *  Show Endpoint in the kubectl get psmdb command output to connect to a MongoDB cluster easily
+
+    *  Store the Operator version as a crVersion field in the deploy/cr.yaml configuration file
+
+    *  Use plain-text passwords instead of base64-encoded ones when creating System Users secrets for simplicity
+
+
+
+
+  maturity: stable
+  version: 1.6.0
+  replaces: ''
```

Comment:
Please point to previous version (percona-server-mongodb-operator.v1.4.0) in replaces field
```
replaces: percona-server-mongodb-operator.v1.4.0
```

Reply:
done